test/cases/testdata/aggregates/test-membership.yaml

```diff
@@ -514,4 +514,17 @@ cases:
   note: aggregates/member+some+key+ref with other variable
   query: data.test.p = x
   want_result:
-  - x: [3, 2, "array"]
\ No newline at end of file
+  - x: [3, 2, "array"]
+- data:
+  modules:
+  - |
+    package test
+    import future.keywords.in
+    p[[k, v]] {
+      some k, v in input with input.foo as "bar"
+    }
+  note: aggregates/member+some+with
+  query: data.test.p = x
+  want_result:
+  - x:
+    - ["foo", "bar"]
```

Comment:
```
 "test/cases/testdata/aggregates/test-membership.yaml is an invalid YAML file"
 ```
 
 uh-huh, ok, I'll check...

Reply:
```
FILE=$(cat test/cases/testdata/aggregates/test-membership.yaml) opa eval -fpretty 'yaml.unmarshal(opa.runtime().env.FILE)' --strict-builtin-errors
```
So that output looks alright to me 🤔

Reply:
The problem seems to have disappeared on its own. Adding a debug step into the check anyways, so we'll have better insights into what's what on failure.